riscv/insts/integer/base/i/ecall/ecall.kt

```diff
@@ -17,6 +17,9 @@ import venusbackend.riscv.insts.dsl.parsers.DoNothingParser
 import venusbackend.riscv.MemorySegments
 import venusbackend.simulator.FilesHandler
 import venusbackend.simulator.Simulator
+import kotlin.js.JSON

```

Comment:
This is only comparable with JavaScript while the backend is supposed to be platform agnostic.